packages/core-utils/src/time.ts

```diff
@@ -131,7 +146,7 @@ export function offsetTime(ms, options) {
  * @param {number} ms epoch time value in milliseconds
  * @returns {string} formatted text representation
  */
-export function formatTime(ms, options) {
+export function formatTime(ms: number, options: TimeOptions): string {
```

Comment:
We should deprecate this method.

Reply:
Deprecated in db1658fad93479daa48f7229371b9aef2200ae86